### PR TITLE
[4.3] Activate all unit tests as default

### PR DESCRIPTION
**Pull request by laoneo**
Activates the extension unit tests as default and does remove some deprecated messages in the tests.

### Before the pr
image

### After the pr
image

### CODE REVIEW

**Comment by Hackwar** [comment]
Thanks for checking this. I doubt that we can merge this without fixing the tests first.

**Comment by heelc29** [comment]
@laoneo see https://github.com/Digital-Peak/joomla-cms/pull/27 for missing namespaces

File: tests/phpunit-appveyor.xml.dist [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <phpunit bootstrap="Unit/bootstrap.php" colors="false">
 	<testsuites>
 		<testsuite name="Unit">
-			<directory suffix="Test.php">./Unit/Libraries</directory>
+			<directory suffix="Test.php">./Unit</directory>
 		</testsuite>
 		<testsuite name="Integration">
 			<directory suffix="Test.php">./Integration/Libraries</directory>
```
Comment (tatankat) [inline tests/phpunit-appveyor.xml.dist]: To enable additional Integration tests and have everywhere the same phpunit config:
```suggestion
			<directory suffix="Test.php">./Integration</directory>
```

**Comment by laoneo** [comment]
@Hackwar can you have a look on this pr, as it is ready now?

**Review by Hackwar — APPROVED** [review]

**Review by rdeutz — APPROVED** [review]

**Comment by wilsonge** [comment]
Think the merge through has gone wrong. You've lost an entire plugin here :)

**Comment by laoneo** [comment]
When 4.3 is up to date, then the pr should be fine again. Do you mean this plugin https://github.com/joomla/joomla-cms/pull/38421/commits/31a2677816a22da4d3f6d831a919efed9b93a139?

**Comment by obuisard** [comment]
Thank you @laoneo for the PR

**Pull request merged**